src/App.js

```diff
@@ -1,12 +1,22 @@
 import React, { Component } from 'react';
+import './App.scss';
+import { Content } from 'carbon-components-react/lib/components/UIShell';
+import TutorialHeader from './components/TutorialHeader';
+import { Route, Switch } from 'react-router-dom';
+import LandingPage from './content/LandingPage';
+import RepoPage from './content/RepoPage';
 
 class App extends Component {
   render() {
     return (
-      <div>
-        Hello Carbon! Well, not quite yet. This is the starting point for the
-        Carbon tutorial.
-      </div>
```

Comment:
Missing `<>` - see https://www.carbondesignsystem.com/tutorial/react-step-1#render-the-header